vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_router_lookahead/config/config.txt

```diff
@@ -27,3 +27,4 @@ pass_requirements_file=pass_requirements_fixed_chan_width.txt
 script_params_common = -starting_stage vpr --route_chan_width 60
 script_params_list_add = --router_lookahead classic
 script_params_list_add = --router_lookahead map
+script_params_list_add = --router_lookahead extended_map
```

Comment:
It would be good to add a test with `--reorder_rr_graph_nodes_algorithm random_shuffle` to make sure there are no dependencies on RR node id order.

Reply:
Added yet another test with the random_shuffle